### PR TITLE
NetSim: Simple use of delete_many

### PR DESCRIPTION
**Pull request by islemaster**
Clients were getting significantly behind in processing messages because they were waiting for sequential delete operations to complete in `destroyEntities`. Here, we change `destroyEntities` to use the new delete_many feature to perform several deletes in a single request, helping clients to catch up if they are behind.

Obviously this is not the _best_ implementation of this idea; I'm in progress on another PR that will also coalesce deletes and let client code "fire and forget" deletes with retries handled nicely inside the `NetSimTable` implementation.  That's a bigger, scarier change - I'd like to get this small change in and see if it improves things first.

### CODE REVIEW

File: apps/src/netsim/NetSimLocalClientNode.js [hunk 1]
```diff
@@ -307,44 +307,6 @@ NetSimLocalClientNode.prototype.getMyRouter = function () {
 };
 
 /**
- * Destroy the local node; performs appropriate clean-up leading up to
- * node destruction.
- * @param {!NodeStyleCallback} onComplete
- */
-NetSimLocalClientNode.prototype.destroy = function (onComplete) {
```
Comment (islemaster) [inline apps/src/netsim/NetSimLocalClientNode.js]: This custom `destroy` method goes away entirely, so the base class `NetSimEntity.destroy` gets called instead.  The additional cleanup is all handled on the server anyway now.

**Comment by islemaster** [comment]
@Hamms please also take a look.

**Comment by philbogle** [comment]
LGTM  Looks very good.

**Comment by Hamms** [comment]
LGTM!

**Pull request merged**